packages/salesforcedx-sobjects-faux-generator/src/messages/index.ts

```diff
@@ -5,39 +5,27 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import * as i18n from './i18n';
-import {
-  BASE_FILE_EXTENSION,
-  BASE_FILE_NAME,
-  Config,
-  DEFAULT_LOCALE,
-  Localization,
-  Message
-} from './localization';
```

Comment:
So you'll find many files updated like this (We have an _almost_ exact copy of this module in every....single...package).  I'll create a ticket if one doesn't exist to move to a single i18n file and just have the messages live in each package. 

Dynamic requires don't play well with bundlers b/c it can't figure out at build time how to pull in the required files. I updated to always just pull in the json files since we only have two and we can adjust in the future if more languages are added.